test/cypress/integration/edit_mode_spec.ts

```diff
@@ -267,12 +252,13 @@ describe('Edit mode', () => {
             const widgetName = 'Backend';
             const widgetId = `testWidget${widgetName}`;
             const url = 'https://this-page-intentionally-left-blank.org/';
-            const verifyRequest = service =>
+            const verifyRequest = (service: string) =>
                 cy.wait(service).then(({ request, response }) => {
                     expect(request.headers).contain({ 'widget-id': widgetId });
-                    expect(response.statusCode).equals(200);
+                    expect(response!.statusCode).equals(200);
```

Comment:
I think it is always cleaner:
```suggestion
                    expect(response?.statusCode).equals(200);
```